target/linux/ar71xx/files/arch/mips/ath79/mach-tl-wr840n-v2.c

```diff
@@ -0,0 +1,127 @@
+/*
+ *  TP-LINK TL-WR841N/ND v9/v11 / TL-WR842N/ND v3
```

Comment:
With your current approach, you are adding two new mach-*.c files which will result in much more more bytes added to the kernel. I'm asking you to drop these files as the same code already exists in other place and there is no reason to duplicate it.